superbench/runner/runner.py

```diff
@@ -173,6 +178,72 @@ def fetch_results(self):    # pragma: no cover
             )
         )
 
+    def __create_results_summary(self):    # pragma: no cover
+        """Create the result summary files."""
+        all_results = list()
+        for node_path in (self._output_path / 'nodes').glob('*'):
+            if not node_path.is_dir():
+                continue
+            results_summary = dict()
+            reduce_ops = dict()
+            file_list = natsorted([str(f) for f in node_path.glob('**/results.json')])
+            file_list = [Path(f) for f in file_list]
+            for results_file in file_list:
+                with results_file.open() as f:
+                    results = json.load(f)
+                    for result in results:
+                        if result['name'] not in results_summary:
+                            results_summary[result['name']] = defaultdict(list)
+                        for metric in result['result']:
+                            results_summary[result['name']][metric].append(result['result'][metric])
+                            reduce_ops[metric] = result['reduce_op'][metric]
+
+            results_summary = self.__merge_all_metrics(results_summary, reduce_ops)
+
+            with (node_path / 'results-summary.json').open(mode='w') as f:
+                json.dump(results_summary, f, indent=2)
+
+            results_summary['node'] = node_path.name
+            all_results.append(results_summary)
+
+        with (self._output_path / 'results-summary.jsonl').open(mode='w') as f:
+            for result in all_results:
+                json.dump(result, f)
+                f.write('\n')
```

Comment:
Do we need this? If so, this format may not be good?

Reply:
I thought we have talked about the summary file for all nodes? one node in json format per line, that is JSON Lines format.
The summary I shared to Akshay is also in JSON Lines format.